data/sql/updates/pending_db_world/rev_1621501368347745883.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1621501368347745883');
+
+UPDATE `creature`
+SET `modelid` = 1744, `spawntimesecs` = 275, `wander_distance` = 5, `curhealth` = 178, `movementtype` = 1
```

Comment:
Why was modelid changed? You didnt mention it in the description so just want to check

Reply:
This was set to 0 before, but since this type of creature only has 1 modelid assigned to it (1744), it would end up being 1744 anyway. I explicitly set it to 1744 to be uniform with the other mobs of the same type